### PR TITLE
Updating json-schema tag

### PR DESCRIPTION
**Pull request by TaylorKrusen**
## What/Why/How?

Updates the json-schema tag to reflect new format. 

## Reference

This change impacts downstream documentation.

## Testing

Previewed change by updating the tag in locally running version of downstream docs.

## Check yourself

- [ ] Code changed? - Tested with redoc/reference-docs/workflows (internal)
- [ ] All new/updated code is covered with tests
- [ ] New package installed? - Tested in different environments (browser/node)

## Security

- [x] Security impact of change has been considered
- [x] Code follows company security practices and guidelines

### CODE REVIEW

File: docs/configuration/apis.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ If per-API and root settings modify the same properties, per-API settings overri
 ## Patterned properties
 
 {% json-schema
-  schemaFilePath="./api.yaml"
+  schema="./api.yaml"
```
Comment (RomanHotsiy) [inline docs/configuration/apis.md:17]: This is not a valid format.

This is the correct one:

```suggestion
  schema={"$ref" : "./api.yaml"}
```

**Review by lornajane — CHANGES_REQUESTED** [review]
I'm currently updating this same issue on the marketing site. @TaylorKrusen could you update your PR according to Roman's advice please?

File: docs/configuration/apis.md (hunk 1 above)
Comment (TaylorKrusen) [reply docs/configuration/apis.md:17]: Oops. Thank you!

**Review by HCloward — APPROVED** [review]
This reflects the new format from Roman's comment.

**Review by lornajane — APPROVED** [review]

**Pull request merged**